deploy/02_sources.ts

```diff
@@ -224,14 +225,34 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
   );
   hre.deployments.log('the future "HolographRoyalties" address is', futureRoyaltiesAddress);
 
+  const network = networks[hre.networkName];
+
+  let tokenAmount: BigNumber = BigNumber.from('100000000');
+  let targetChain: BigNumber = BigNumber.from(network.chain);
```

Comment:
is this so it can run on localhost actually?

Reply:
yep, this is to allow localhost and testnets to work still, while mainnet is strictly restricted to only minting on ethereum.